PRIYA/index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">

```

Comment:
using bootstrap would have saved you a lot of time , and the styling of buttons could have been much better